src/app-layout/__integ__/toolbar-tooltips/app-layout-toolbar-split-panel-trigger-tooltip.test.ts

```diff
@@ -0,0 +1,130 @@
+  );
+  const appliedThemeStyles = toolbarStyles; //use visualRefreshStyles when theme === 'refresh'
+
+  describe.each(['desktop', 'mobile'] as const)('%s', size => {
+    const drawersTriggerContainerClassKey = `drawers-${size === 'desktop' ? 'desktop' : 'mobile'}-triggers-container`;
+    const drawerIdsToTest = size === 'mobile' ? mobileDrawerTriggerIds : drawerIds; //toolbarDrawerIds;
+    const firstDrawerTriggerSelector = wrapper.findDrawerTriggerById(drawerIdsToTest[0]).toSelector();
+    const splitPanelTriggerSelector = wrapper.findSplitPanelOpenButton().toSelector();
+    const tooltipSelector = wrapper.findDrawerTriggerTooltip().toSelector();
+    const expectedTooltipText = 'Open panel';
+
+    describe.each(['bottom', 'side'] as const)('splitPanelPosition=%s', splitPanelPosition => {
+      test(
+        'Shows tooltip correctly on split panel trigger for mouse interactions',
+        setupTest({ theme, size, splitPanelPosition }, async (page: AppLayoutDrawersPage) => {
+          await expect(page.getElementsCount(tooltipSelector)).resolves.toBe(0);
+          await page.hoverElement(splitPanelTriggerSelector);
+          await expect(page.getText(tooltipSelector)).resolves.toBe(expectedTooltipText);
+          await expect(page.getElementsCount(tooltipSelector)).resolves.toBe(1);
+          await page.hoverElement(wrapper.findNavigationToggle().toSelector());
+          await expect(page.isExisting(tooltipSelector)).resolves.toBe(false);
+        })
+      );
+
+      test(
+        'Shows tooltip correctly on split panel trigger for mouse interactions during split-panel actions',
+        setupTest({ theme, size, splitPanelPosition }, async (page: AppLayoutDrawersPage) => {
+          await expect(page.getElementsCount(tooltipSelector)).resolves.toBe(0);
+          await expect(
+            page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)
+          ).resolves.toBeTruthy();
+          await page.click(splitPanelTriggerSelector);
+          await expect(page.isExisting(tooltipSelector)).resolves.toBe(false);
+          await page.click(splitPanelTriggerSelector); //this could change witf focus fixes
+          await expect(page.isExisting(tooltipSelector)).resolves.toBe(false);
+          await page.hoverElement(firstDrawerTriggerSelector);
+          await expect(page.getElementsCount(tooltipSelector)).resolves.toBe(1);
+          await page.hoverElement(splitPanelTriggerSelector);
+          await expect(page.getElementsCount(tooltipSelector)).resolves.toBe(1);
+        })
+      );
+
+      test(
+        'Removes tooltip from split panel trigger on escape key press after showing from mouse hover',
+        setupTest({ theme, size, splitPanelPosition }, async (page: AppLayoutDrawersPage) => {
+          await expect(page.getElementsCount(tooltipSelector)).resolves.toBe(0);
+          await expect(
+            page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)
```

Comment:
Do you need to check existence of this button here? I don't really get how it's connected with other testing conditions